src/go/plugin/go.d/collector/dcgm/integrations/nvidia_data_center_gpu_manager_dcgm.md

```
@@ -242,11 +244,11 @@ The following alerts are available:
 
 | Alert name  | On metric | Description |
 |:------------|:----------|:------------|
-| [ dcgm_gpu_xid_errors ](https://github.com/netdata/netdata/blob/master/src/health/health.d/dcgm.conf) | dcgm.gpu.reliability.xid | NVIDIA driver reported GPU XID error on GPU ${label:gpu} |
-| [ dcgm_gpu_row_remap_failure ](https://github.com/netdata/netdata/blob/master/src/health/health.d/dcgm.conf) | dcgm.gpu.reliability.row_remap_status | GPU row remapping failed on GPU ${label:gpu} |
-| [ dcgm_gpu_uncorrectable_remapped_rows ](https://github.com/netdata/netdata/blob/master/src/health/health.d/dcgm.conf) | dcgm.gpu.reliability.row_remap_events | Uncorrectable remapped rows increased on GPU ${label:gpu} |
-| [ dcgm_gpu_power_violation ](https://github.com/netdata/netdata/blob/master/src/health/health.d/dcgm.conf) | dcgm.gpu.throttle.violations | Power throttling detected on GPU ${label:gpu} |
-| [ dcgm_gpu_thermal_violation ](https://github.com/netdata/netdata/blob/master/src/health/health.d/dcgm.conf) | dcgm.gpu.throttle.violations | Thermal throttling detected on GPU ${label:gpu} |
+| [ dcgm_gpu_xid_errors ](https://github.com/netdata/netdata/blob/master/src/health/health.d/dcgm.conf) | dcgm.gpu.reliability.xid | NVIDIA driver reported a GPU XID error (metric ${label:chart_context}). |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)